runbot_travis2docker/models/runbot_repo.py

```diff
@@ -40,3 +40,4 @@ def weblate_validation(self):
             json = response.json()
             if 'projects' not in json:
                 raise ValidationError(_('Response json bad formated'))
+        raise Warning(_('Connection with weblate successful'))
```

Comment:
I'm worried because the raise run a rollback so if I use this method with other ones we will have information deleted.
Is there other way to set this information to user?
(Maybe a extra field, maybe chatter, maybe remove it...  I don't know)